operator/templates/stateful-set.yaml

```diff
@@ -133,6 +135,63 @@ spec:
             - name: jvm-options
               mountPath: /etc/cassandra/jvm.options
               subPath: jvm.options
+        {{ if eq .Params.METRICS_ENABLED "true" }}
+        - name: metrics
+          imagePullPolicy: Always
+          image: viivek46/cassandra:3.11.4
+          securityContext:
+            capabilities:
+              add:
+                # TODO(mpereira): add an operator parameter for "memory lock" so
+                # that this capability and a respective Docker image with
+                # `setcap`ed binaries are conditionally used.
+                - IPC_LOCK
+                # FIXME(mpereira): might not be necessary given DCOS-59219.
+                - SYS_RESOURCE
+          command:
+            - bash
+            - -c
+          args:
+            - cd /etc/cassandra;
+              curl -L -O https://github.com/criteo/cassandra_exporter/releases/download/2.2.1/cassandra_exporter-2.2.1-all.jar;
+              java -jar cassandra_exporter-2.2.1-all.jar cassandra-exporter-config.yml;
```

Comment:
Using a Docker image based on the one available in the cassandra_exporter repository (as suggested in a previous comment) will allow the container `cmd` and `args` to be much simpler and not require downloading the JAR at runtime.